packages/story-editor/src/components/panels/document/publish/publish.js

```diff
@@ -250,22 +297,23 @@ function PublishPanel() {
                     node.focus();
                   }
                 }}
-                value={publisherLogo.url}
-                onChange={handleChangePublisherLogo}
-                onChangeErrorText={publisherLogoErrorMessage}
-                title={__('Select as publisher logo', 'web-stories')}
-                buttonInsertText={__('Select as publisher logo', 'web-stories')}
-                type={allowedImageMimeTypes}
-                ariaLabel={__('Publisher Logo', 'web-stories')}
-                variant={MEDIA_VARIANTS.CIRCLE}
-                canUpload={hasUploadMediaAction}
               />
             </MediaWrapper>
             <LabelWrapper>
               <Label>{__('Publisher Logo', 'web-stories')}</Label>
-              <Required />
+              <Row>
+                <Required />
+                <Link
+                  rel="noopener noreferrer"
+                  target="_blank"
+                  href={settingsLink}
+                  size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}
+                >
+                  {__('Manage', 'web-stories')}
+                </Link>
```

Comment:
Should it open in a new tab or do we want to rely on the `UnsavedChangesWarning`? also, is there a better way than hardcoding the URL like this?